dts-generator/src/main/java/com/telerik/dts/DtsApi.java

```diff
@@ -498,8 +504,7 @@ private void writeMethods(Set<String> methodsSet) {
     }
 
     private void cacheMethodBySignature(Method m) {
-        mapNameMethod = new HashMap<String, Method>();
-        String currMethodSig = getMethodFullSignature(m);
+        String currMethodSig = m.getName();//getMethodFullSignature(m);
```

Comment:
change variable name to better describe its use and remove the comment if it's no longer necessary